src/CommitmentsV2/SFA.DAS.CommitmentsV2.Messages/Events/EntityStateChangedEvent.cs

```diff
@@ -30,6 +30,7 @@ public enum UserAction
         DeleteDraftApprenticeship,
 		DeleteCohort,
         ApproveTransferRequest,
-        RejectTransferRequest
+        RejectTransferRequest,
+        CompletionPayment
```

Comment:
I'd prefer "Complete" - we have a pattern of naming the action after the method. This is more elegant as it need make no mention of payment, which is merely the trigger for our state change.